docs/api.md

```diff
@@ -325,6 +327,15 @@ Note that some endpoints do not require authorization:
 * [`POST /api/register`](#post-apiregister)
 * [`GET /api/session/:sessionID`](#get-apisessionsessionid)
 
+### Authentication:
+
+When a request is made to the API, the server searches for a session ID given in the request using:
+* `sessionID` in POST data
+* `?sessionID` in query string
+* `X-Session-ID` header
+
+If the server requires [authorization](#authorization) and the session ID could not be found or pointed to an unauthenticated user, the request is immediately terminated with a 403 and error message. We recommend simply sending the `X-Session-ID` header with _all_ requests, even on non-authorization-requiring servers.
```

Comment:
I don't like "we" - this should be `it's recommended to` 😃

Reply:
`It's probably simpler to send` thanks